lib/backend/etcdv3/etcdv3.go

```diff
@@ -124,6 +124,11 @@ func (c *etcdV3Client) Create(ctx context.Context, d *model.KVPair) (*model.KVPa
 		return existing, cerrors.ErrorResourceAlreadyExists{Identifier: d.Key}
 	}
 
+	v, err := model.ParseValue(d.Key, []byte(value))
+	if err != nil {
+		cerrors.PanicIfErrored(err, "Unexpected error parsing stored datastore entry: %v", value)
```

Comment:
You don't need the if `err != nil` check to use PanicIfErrored

Reply:
good to know